doc/api/cli.md

```diff
@@ -288,6 +288,17 @@ Path to the file used to store the persistent REPL history. The default path is
 to an empty string (`""` or `" "`) disables persistent REPL history.
 
 
+### `NODE_TTY_UNSAFE_ASYNC=1`
+<!-- YAML
+added: REPLACEME
+-->
+
+When set to `1` writes stdout and stderr will be non-blocking and async when
+outputting to a TTY on platforms which support async stdio.
```

Comment:
Can we expand on this with specifics? Do we have an accurate understanding of Windows now (I can't keep track of how many times both sides have been asserted for Windows)?